json-schemas/amrWorkflowResults.json

```diff
@@ -33,7 +33,7 @@
                 }
             }
         },
-        "report_table_data": {
+        "amr_hit": {
```

Comment:
Align to schema returned by resolver.  This was preventing the tests from running, because the schema could not be generated